floating_text/floating_text.py

```diff
@@ -0,0 +1,188 @@
+                       "**** ",
+                       "*   *",
+                       "**** " ],
+
+               "C" : [ " ****",
+                       "*    ",
+                       "*    ",
+                       "*    ",
+                       " ****" ],
+
+               "D" : [ "**** ",
+                       "*   *",
+                       "*   *",
+                       "*   *",
+                       "**** " ],
+
+               "E" : [ "*****",
+                       "*    ",
+                       "*****",
+                       "*    ",
+                       "*****" ],
+
+               "F" : [ "*****",
+                       "*    ",
+                       "*****",
+                       "*    ",
+                       "*    " ],
+
+               "G" : [ " ****",
+                       "*    ",
+                       "* ***",
+                       "*   *",
+                       " ****" ],
+
+               "H" : [ "*   *",
+                       "*   *",
+                       "*****",
+                       "*   *",
+                       "*   *" ],
+
+               "I" : [ "*****",
+                       "  *  ",
+                       "  *  ",
+                       "  *  ",
+                       "*****" ],
+
+               "J" : [ "*****",
+                       "   * ",
+                       "   * ",
+                       "*  * ",
+                       "**** " ],
+
+               "K" : [ "*   *",
+                       "*  * ",
+                       "***  ",
+                       "*  * ",
+                       "*   *"  ],
+
+               "L" : [ "*    ",
+                       "*    ",
+                       "*    ",
+                       "*    ",
+                       "*****" ],
+
+               "M" : [ "*   *",
+                       "** **",
+                       "* * *",
+                       "*   *",
+                       "*   *" ],
+
+               "N" : [ "*   *",
+                       "**  *",
+                       "* * *",
+                       "*  **",
+                       "*   *" ],
+
+               "O" : [ " *** ",
+                       "*   *",
+                       "*   *",
+                       "*   *",
+                       " *** " ],
+
+               "P" : [ "**** ",
+                       "*   *",
+                       "**** ",
+                       "*    ",
+                       "*    " ],
+
+               "Q" : [ " *** ",
+                       "**  *",
+                       "* * *",
+                       " *** ",
+                       "   * " ],
+
+               "R" : [ "**** ",
+                       "*   *",
+                       "**** ",
+                       "*  * ",
+                       "*   *"  ],
+
+               "S" : [ " ****",
+                       "*    ",
+                       " *** ",
+                       "    *",
+                       "**** " ],
```

Comment:
What about numbers ?

Reply:
And remove the cls not found command.

Its obstructing the view

Reply:
The program is just for English alphabets.

Reply:
Okay, I will fix that.